api/src/components/coauthor/controller.ts

```diff
@@ -1,12 +1,23 @@
 import * as coAuthorService from 'coauthor/service';
-import * as email from 'email';
 import * as I from 'interface';
+import * as email from 'email';
 import * as response from 'lib/response';
 import * as publicationService from 'publication/service';
 
```

Comment:
too many line breaks, need to run prettier, prob same on other files, will let this comment stand for all :)